crates/sdk/Cargo.toml

```diff
@@ -13,37 +13,40 @@ name = "ndc_hub_example"
 path = "bin/main.rs"
 
 [features]
+default = ["native-tls", "ndc-test"]
+
+native-tls = ["ndc-client/native-tls", "ndc-test/native-tls", "reqwest/native-tls"]
+rustls = ["ndc-client/rustls", "ndc-test/rustls", "reqwest/rustls"]
+
 ndc-test = ["dep:ndc-test"]
-default = ["ndc-test"]
 
 [dependencies]
-ndc-client = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.0" }
-ndc-test = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.0", optional = true }
-
-async-trait = "^0.1.74"
-axum = "^0.6.20"
-axum-extra = "^0.8.0"
-bytes = "1.5.0"
-clap = { version = "^4.4.7", features = ["derive", "env"] }
-http = "^0.2"
+ndc-client = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.1" }
+ndc-test = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.1", optional = true }
+
+async-trait = "0.1.79"
```

Comment:
How do these version upgrades come about? Is it just `cargo update` or somesuch?

Reply:
`cargo upgrade`. I was upgrading `opentelemetry` and friends to match ndc-client and figured I might as well do the rest (except for upgrades to the HTTP stack, which are incompatible).